src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2680,6 +2680,8 @@ internal static BaseParameterListSyntax GetParameterList(SyntaxNode declaration)
                     return ((ParenthesizedLambdaExpressionSyntax)declaration).ParameterList;
                 case SyntaxKind.SimpleLambdaExpression:
                     return SyntaxFactory.ParameterList(SyntaxFactory.SingletonSeparatedList(((SimpleLambdaExpressionSyntax)declaration).Parameter));
+                case SyntaxKind.LocalFunctionStatement:
+                    return ((LocalFunctionStatementSyntax)declaration).ParameterList;
```

Comment:
I don't know why all the other diffs show up on github, but I actually only added these two lines to CSharpSyntaxGenerator.cs
![2018-09-12 11_07_28-roslyn - microsoft visual studio](https://user-images.githubusercontent.com/24472128/45414722-4f73f700-b67c-11e8-9535-a9a76764c8b4.png)